src/tools/crossgen2/ILCompiler.DependencyAnalysisFramework/DependencyAnalyzer.cs

```diff
@@ -250,13 +250,16 @@ private void ProcessMarkStack()
 
         public override void ComputeMarkedNodes()
         {
+            DependencyAnalysisPerfEventSource.Log.GraphProcessingStart();
```

Comment:
Nit: we could have left both of these named `PerfEventSource`. It's more concise. Once the one in `ILCompiler.DependencyAnalysisFramework` is marked `internal`, the names won't clash.

Reply:
Ah yes I considered both options but I didn't know which to choose. Changed.